agent/src/agent/pipeline/config/stages/expression_evaluator.py

```diff
@@ -1,22 +1,22 @@
 import csv
 
 from .base import Stage
-from agent.pipeline.pipeline import TimestampType
 from agent.pipeline.config.expression_parser import condition
+from agent.pipeline import pipeline
 
 
 def get_value(path, expr) -> dict:
     return {'fieldToSet': path, 'expression': '${' + expr + '}'}
 
 
-def get_convert_timestamp_to_unix_expression(timestamp_type: TimestampType, value, timestamp_format):
-    if timestamp_type == TimestampType.STRING:
+def get_convert_timestamp_to_unix_expression(timestamp_type: pipeline.TimestampType, value, timestamp_format):
+    if timestamp_type == pipeline.TimestampType.STRING:
         return f"time:dateTimeToMilliseconds(time:extractDateFromString({value}, '{timestamp_format}'))/1000"
     elif timestamp_type == TimestampType.UTC_STRING:
```

Comment:
why not pipeline.TimestampType.UTC_STRING?

Reply:
I don't know :) because it used to be TimestampType.STRING, not TimestampType.UTC_STRING, I just added 'pipeline.'